### ISSUE TITLE
Серверная часть

### ISSUE THREAD

**Issue by ghost** [LeagueOfDevelopers/Today_User#7]
Поставить некоторую базу данных, написать серверную часть ответа на запросы, поднять это счастье

**Comment by Sovent** [comment]
Так ты реляционную базу хочешь или XML? У тебя же таска 5 есть

**Comment by ghost** [comment]
Так базы есть две - первая реляционная(на сервере) вторая локальная, на пк - в xml.

**Comment by Sovent** [comment]
Будет какая-то синхронизация?

**Comment by ghost** [comment]
Да. При загрузке приложения оно будет собирать идентификаторы локальных сообщений из  своей базы, отправлять их на сервер, а сервер будет возвращать недостающие.

**Comment by Sovent** [comment]
Можно индексировать сообщения  от 1 инкрементно и отправлять для синхронизации самый большой индекс, сервер вернёт сообщения с большим индексом. Либо присваивать каждому клиенту некий ID, сохранять его в базе на сервере, хранить дату его последней синхронизации и каждый раз выдавать сообщения, созданные позже этой даты. Энивей, этот таск не в Today_User.
